synfig-studio/plugins/lottie-exporter/lottie-exporter.py

```diff
@@ -20,6 +20,53 @@
 import settings
 import argparse
 
+def calc_font_data(lottie,layer):
+    """
+    Calculates the font data used in the outermost canvas
+
+    Args:
+        lottie (dict)       : Outermost dictionary used in Lottie file
+        layer  (common.Layer.Layer) : Synfig format text layer
+
+    Returns:
+        (None)
+    """
+    default_list = {
+        "origin": 0,
+        "fPath": "",
+        "fClass": "",
+        "fFamily": "",
+        "fWeight": "",
+        "fName": "",
+        "ascent": 75.9994506835938
+      }
+
+    style_dict = {0:'Regular',1:'Regular',2:'Italic'}
```

Comment:
Why here two 'Regular' styles?
Also maybe add a spaces for better reading, like:
```
style_dict = {0:'Regular', 1:'Regular', 2:'Italic'}
```

Reply:
Synfig also has oblique style which is not supported by Lottie so, for that I'm defaulting to Regular style.

Reply:
Can you add this comment to code?

Reply:
Yup, would be nice to have this as a comment.

Reply:
Done ^__^